Dockerfile

```diff
@@ -8,6 +9,14 @@ COPY src/main/resources/static/fonts/*.ttf /usr/share/fonts/opentype/noto/
 #COPY src/main/resources/static/fonts/*.otf /usr/share/fonts/opentype/noto/
 COPY build/libs/*.jar app.jar
 
+# Copy files result to a new image.
+# This saves a lot of disk space.
+FROM alpine:3.21.2@sha256:56fa17d2a7e7f168a043a2712e63aed1f8543aeafdcee47c58dcffe38ed51099
+COPY --from=copy_build_files scripts /scripts
+COPY --from=copy_build_files pipeline /pipeline
+COPY --from=copy_build_files src/main/resources/static/fonts/*.ttf /usr/share/fonts/opentype/noto/
```

Comment:
`COPY --from=copy_build_files /usr/share/fonts/opentype/noto/ /usr/share/fonts/opentype/noto/`

Reply:
es wird von `src/main/resources/static/fonts/*.ttf` zu `/usr/share/fonts/opentype/noto/` kopiert, dasher kann es nicht in `src/main/resources/static/fonts/*.ttf` vorhanden sein